apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -305,6 +310,17 @@ int _wt_get_scanned_list(wifi_manager_scan_info_s *slist, char *ssid,
 	}
 	return -1;
 }
+
+wifi_manager_ap_auth_type_e wm_get_auth_type(const char *method)
+{
+	return _wt_get_auth_type(method);
+}
+
+wifi_manager_ap_crypto_type_e wm_get_crypto_type(const char *method)
+{
+	return _wt_get_crypto_type(method);
+}
```

Comment:
why did you define wm_get_crypto_type not to call _wt_get_crypto_type directly?

Reply:
Since _wt_get_crypto_type is static and local to wm_test hence used wm_get_crypto_type as external function

Reply:
how about to move _wt_get_auth_type to wm_test_utils.c